content/governance.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Governance
+layout: page
+
+---
+
+# Governance
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. Common terminology used in this governance document are listed below:
+
+* **Team members**: Any members of the private [prometheus-team][team] Google group.
+
+* **Maintainers**: Maintainers lead an individual project or parts thereof ([MAINTAINERS.md][maintainers.md]).
+
+* **Projects**: Any repository in the Prometheus [GitHub organization][gh].
```

Comment:
Nit: Strictly reading, this implies a single repository is called _Projects_. Either write "Repositories in the Prometheus [GitHub organization][gh]." or be more explicit: "A single repository in the Prometheus [GitHub organization][gh] is referred to as a Project."